### PR TITLE
`farish.json`

### PR DESCRIPTION
**Pull request by iamfarishirfan**
#415

### CODE REVIEW

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
You need a separate file for this

**Comment by iamfarishirfan** [comment]
> You need a separate file for this

What do you mean by "this"?

**Comment by WilliamDavidHarrison** [comment]
> > You need a separate file for this
> 
> What do you mean by "this"?

The acme challenge.

**Comment by iamfarishirfan** [comment]
Oh, in that case, the `name` value under `target.CNAME` is my requested subdomain?

**Comment by WilliamDavidHarrison** [comment]
> Oh, in that case, the `name` value under `target.CNAME` is my requested subdomain?

Yes

**Comment by iamfarishirfan** [comment]
I've changed that.

File: sub-logs/farish.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  "owner": {
+    "username": "iamfarishirfan",
+    "email": "hello.farish.irfan@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "farish", 
+      "value": "700b3eafa2ce95949fe9e0e2._acme.deno.dev."
```
Comment (DEV-DIBSTER) [inline sub-logs/farish.json:12]: You don’t need a dot at the end of the value.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by DEV-DIBSTER** [comment]
Your CNAME record isn’t returning anything, are you sure it’s valid?

**Comment by iamfarishirfan** [comment]
> Your CNAME record isn’t returning anything, are you sure it’s valid?

Yes.
image

**Comment by iamfarishirfan** [comment]
How long will it take to get my PR merged?

**Comment by Tweak4141** [comment]
From your screenshot, you will most likely need an A record added. If you wish to just merge, please let me know,'

**Comment by Tweak4141** [comment]
> From your screenshot, you will most likely need an A record added. If you wish to just merge, please let me know.
A's name value should be `farish`
CNAME's name value should be the acme challenge.

**Comment by iamfarishirfan** [comment]
Alright, I'll update it a bit later.

**Review by WilliamDavidHarrison — DISMISSED** [review]

**Review by WilliamDavidHarrison — DISMISSED** [review]

**Review by WilliamDavidHarrison — DISMISSED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
@Tweak4141 `_acme-challenge` is an invalid domain, do we accept the domains or not?

**Comment by Tweak4141** [comment]
It can be accepted if the acme challenge is restricted to the subdomain.

**Comment by WilliamDavidHarrison** [comment]
> It can be accepted if the acme challenge is restricted to the subdomain.

It is.

**Comment by iamfarishirfan** [comment]
I've talked to my provider and he says I only required to add `CNAME` with `farish` instead of `_acme-challenge`

**Comment by Tweak4141** [comment]
Alright, sounds good. Please make your changes, then let us know when you are done.

**Comment by iamfarishirfan** [comment]
Should be good now @Tweak4141

File: sub-logs/farish.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  "owner": {
+    "username": "iamfarishirfan",
+    "email": "hello.farish.irfan@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "farish",
+      "value": "700b3eafa2ce95949fe9e0e2._acme.deno.dev"
```
Comment (DEV-DIBSTER) [inline sub-logs/farish.json:12]: I'm unable to load the CNAME record, as it's doesn't load.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: sub-logs/farish.json (hunk 2 above)
Comment (iamfarishirfan) [reply sub-logs/farish.json:12]: I'm just following the instructions provided by my provider. Can you merge the PR first and if it doesn't work, at least I have proof it doesn't work which I can show to him.

**Pull request merged**